scopes/lanes/ui/menus/use-lanes-menu.tsx

```diff
@@ -15,10 +15,15 @@ export type LaneImportContentProps = {
   host: LanesHost;
 } & React.HTMLAttributes<HTMLDivElement>;
 
-export function UseLaneMenu({ host }: { host: LanesHost }) {
-  const lanesContext = useLanesContext();
-  if (!lanesContext?.viewedLane) return null;
-  const { viewedLane, currentLane } = lanesContext;
+export function UseLaneMenu({
+  host,
+  viewedLane,
+  currentLane,
+}: {
+  host: LanesHost;
+  viewedLane: LaneModel;
```

Comment:
I don't have the LaneModel so would prefer to pass more specific info here. i noticed it only uses the id. am i right?

Reply:
it uses id and name. I updated it accept `{id: string, name: string`} instead. 
it uses the name in case you are on a workspace, so it prompts you correctly on how to switch to the lane